docs/projects/merge-sort/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Merge Sort in Every Language layout: default
+date: 2018-11-29
+last-modified: 2018-11-29
+featured-image:
+tags: [merge-sort]
+authors:
+  - auroq
+---
+
+In this article, we'll tackle Merge Sort, an efficient sorting algorithm.
+
+## Description
+
+Merge sort is an algorithm that works by dividing a list into smaller lists.
+It continues dividing until each list only has a single element in it
+because lists of a single element are by definition already sorted.
+It then merges each sublist in a sorted fashion until they all become a single sorted list.
+
+Step by step the process is:
+
+1. Divide the sorted list into lists of 1 element.
```

Comment:
I think this list might render a littler odd. At least it does in GitHub markdown. You may need to indent the bullets further.

Reply:
I fixed it!